src/rosdiscover/observer/ros1.py

```diff
@@ -94,3 +97,17 @@ def _transform_state_to_nodeinfo(self, state: SystemState) -> Collection[NodeInf
                         reorganized_nodes[node_name].provides.add(service)
 
         return list(reorganized_nodes.values())
+
+    def execute_script(self, path_on_host: str) -> Popen:
+        if not os.path.exists(path_on_host):
+            raise FileNotFoundError(f"'{path_on_host}' not found.")
+        assert self._app_instance is not None
+
+        path_on_container = os.path.join('/tmp', os.path.basename(path_on_host))
```

Comment:
We could use this to be safer: https://github.com/ChrisTimperley/dockerblade/blob/master/src/dockerblade/files.py#L540

Reply:
Done.